templates/archive.html

```diff
@@ -34,4 +34,4 @@
 </div>
 <!-- /wp:group -->
 
-<!-- wp:template-part {"slug":"footer","theme":"course","tagName":"footer"} /-->
```

Comment:
I remove this because it is breaking on my machine.
We can have another Pr only to add it on all files.